modules/private-shared-lib/tsconfig.json

```
@@ -1,10 +1,18 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-	"rootDir": "./src",
-	"outDir": "./dist"
+    "rootDir": "./src",
+    "outDir": "./dist",
+    "paths": {
```

Is `baseUrl` not required here?

```suggestion
    "baseUrl": ".",
    "paths": {
```

---

Thanks, I forgot to add it back.